Domain/Models/Seats/SeatEquipments.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Models.Seats
+{
+    public class SeatEquipments : BaseEntity
+    {
+        public int SeatId { get; set; }
+        public int  EquipmentId { get; set; }
+
+        public Seat seat { get; set; }
```

Comment:
All property names should start with Upper-case

Reply:
I updated to Upper-case, now all of the properties' name start with uppercase letter